src/main/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -90,9 +90,30 @@ import com.pubnub.api.presence.Presence
 import com.pubnub.api.presence.eventengine.data.PresenceData
 import com.pubnub.api.presence.eventengine.effect.effectprovider.HeartbeatProviderImpl
```

Comment:
we could update the spec, although AutoCloseable and Close is a Java thing. 
For subscription, close == unsubscribe
For SubscriptionSet it's a bit more tricky, because it actually needs a proper `close()`